react-native/react/actions/tracker.js

```diff
@@ -147,45 +145,27 @@ export function onCloseFromHeader (username: string): Action {
   }
 }
 
-function loadUserInfo (uid: any, username:string): (dispatch: Dispatch) => void {
-  return dispatch => {
-    engine.rpc('user.loadUncheckedUserSummaries', {uids: [uid]}, {}, (error: ?any, response: Array<UserSummary>) => {
-      if (error) {
-        console.log(error)
-        return
-      }
-
-      const onlyUser: ?UserSummary = response[0]
-      if (!onlyUser) {
-        console.log('Did not get back a user summary')
-        return
-      }
-
-      dispatch(updateUserInfo({
-        fullname: onlyUser.fullName,
-        avatar: onlyUser.thumbnail,
-        // TODO: get this data from somewhere
-        location: '', // TODO: get location data
-        followersCount: -1,
-        followingCount: -1,
-        followsYou: false
-      }, username))
-    })
-  }
-}
-
-function updateUserInfo (userInfo: UserInfo, username: string): Action {
+function updateUserInfo (userCard: UserCard, username: string, getState: Function): Action {
```

Comment:
ah you're right sorry, then just update the getState to be `getState: () => {tracker: RootTrackerState}`